app/main.ipynb

```diff
@@ -403,23 +541,65 @@
     "result"
```

Comment:
The result variable is not updated after calling the function. Ensure the function's return value is assigned to the result variable.